content/support-matrix/delfin/_index.md

```diff
@@ -17,10 +17,15 @@ Delfin has standardized common API services.
 | Dell EMC                                            | VNX (Block)               | 4.30 or later                      | CLI                     | Block   | √                        | √            | √   | √          | √    | √    | NA    | NA    | NA          | NA    | √                    | √         | ×           | ×               | ×                   | 1\. Before connecting a Dell EMC VNX Block storage device, install Navisphere Secure CLI on delfin installed node.                                                                                      |   |
```

Comment:
According to the code implementation, I find that the statistics of the vmax device are different from those of the unity device. The red part indicates the result after the change. Please check it.
![image](https://user-images.githubusercontent.com/59107077/157819340-a98d99f2-7cc2-46aa-81ba-635c301286f0.png)